backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,225 @@
+
+        if options["project_key"]:
+            queryset = Project.objects.filter(key=options["project_key"])
+        elif options["all"]:
+            queryset = Project.objects.all()
+        else:
+            queryset = Project.objects.filter(is_active=True)
+
+        if not (total_projects := queryset.count()):
+            self.stdout.write("No projects found to process")
+            return
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)